src/main/java/frc/robot/commands/GetCenterNoteX.java

```diff
@@ -57,7 +57,7 @@ public GetCenterNoteX(Translation2d targetNote, DriveTrain driveTrain, NoteVisio
 
             // wait up to 1 second to suck the Note in all the way
             // new WaitUntilCommand(intake::hasNote).withTimeout(1),
-            new WaitCommand(1),
+            new WaitCommand(0.5),
```

Comment:
We might want to make this a named constant. We could do some testing how much to reduce it.